pkg/aspect/query/query_test.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright © 2021 Aspect Build Systems Inc
+
+Not licensed for re-use.
+*/
+
+package query_test
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/gomega"
+
+	"aspect.build/cli/pkg/aspect/query"
+	"aspect.build/cli/pkg/bazel/mock"
+	"aspect.build/cli/pkg/ioutils"
+)
+
+type confirm struct{}
+
+func (p confirm) Run() (string, error) {
+	return "", nil
+}
+
+func TestQuery(t *testing.T) {
```

Comment:
We could use some more tests that go over more code paths.

Reply:
Agreed. It needs some more tests. I got a bit hung up trying to figure out how to mock the inputs to `promptui`. (As several of the code paths rely on its inputs)